src/kixi/stats/distribution.cljc

```diff
@@ -4,6 +4,12 @@
             [kixi.stats.protocols :as p :refer [sample-1 sample-n sample-frequencies]]
             [clojure.test.check.random :refer [make-random rand-double rand-long split split-n]]))
 
+;;;; Assert helpers
+
+(defn ^:no-doc non-neg?
+  [x]
+  (complement neg?))
```

Comment:
`x` isn't being used in the function body, so the function returns `(complement neg?)` without applying it. I guess this would mean it always returns a truthy value. `(def non-neg? (complement neg?))` would work, as would `(defn non-neg? [x] (not (neg? x)))`.

Reply:
Sorry. Embarrassing. Fixed.